app/src/lib/enable-source-maps.ts

```diff
@@ -0,0 +1,60 @@
+import * as Path from 'path'
+import * as Fs from 'fs'
+
+const fileUriToPath = require('file-uri-to-path') as (uri: string) => string
+const sourceMapSupport = require('source-map-support')
+
+// Can't use in dev builds because if the fs module is available source-maps-support always uses that
```

Comment:
Isn't the `fs` module available in prod too? I'm not sure I understand this comment 🤔